src/directives/internal/class.js

```diff
@@ -77,3 +73,39 @@ function stringToObject (value) {
   }
   return res
 }
+
+/**
+ * Add or remove a class/classes on an element
+ *
+ * @param {Element} el
+ * @param {String} key The class name. This may or may not
+ *                     contain a space character, in such a
+ *                     case we'll deal with multiple class
+ *                     names at once.
+ * @param {Boolean} add
```

Comment:
instead of passing a boolean, we can just pass in `addClass` or `removeClass` and call it directly.